packages/runtime/test/integration/c2c-call/c2c-call.ts

```diff
@@ -292,7 +292,10 @@ describe("C2C call", function () {
 		it("Should not support other inner tx appl(not include appcall)", () => {
 			assert.doesNotThrow(() => runtime.executeTx([execParams]));
 			assert.isTrue(
-				(console["warn"] as any).calledWith("Only supports application call in this version")
+				(console["warn"] as any).calledWith(
+					`\x1b[33m%s\x1b[0m`,
```

Comment:
Could you explain what it does?

Reply:
this is a color formatter. Please don't use it manually, we have a package for that. For example: `chalk.greenBright`. Please see files in `algob/src/internal/cli`